executor/split.go

```diff
@@ -777,19 +777,23 @@ func (d *regionKeyDecoder) decodeRegionKey(key []byte) string {
 
 func getRegionMeta(tikvStore helper.Storage, regionMetas []*tikv.Region, uniqueRegionMap map[uint64]struct{},
 	tablePrefix, recordPrefix, indexPrefix []byte, physicalTableID, indexID int64,
-	hasUnsignedIntHandle bool) ([]regionMeta, error) {
-	regions := make([]regionMeta, 0, len(regionMetas))
+	hasUnsignedIntHandle bool) ([]showTableRegionRowItem, error) {
+	regions := make([]showTableRegionRowItem, 0, len(regionMetas))
 	for _, r := range regionMetas {
 		if _, ok := uniqueRegionMap[r.GetID()]; ok {
 			continue
 		}
 		uniqueRegionMap[r.GetID()] = struct{}{}
-		regions = append(regions, regionMeta{
-			region:   r.GetMeta(),
-			leaderID: r.GetLeaderPeerID(),
-			storeID:  r.GetLeaderStoreID(),
+		regions = append(regions, showTableRegionRowItem{
+			regionMeta: regionMeta{
+				region:   r.GetMeta(),
+				leaderID: r.GetLeaderPeerID(),
+				storeID:  r.GetLeaderStoreID(),
+			},
+			physicalID: physicalTableID,
```

Comment:
Hi @lcwangchao, I agree with what you mentioned of avoiding to add `showTableRegionRowItem` to exist functions. But I need this physicalId in each regions to find the correct scheduling info for each region later. Can I add physicalID to regionMeta or if there's any other better way?

Reply:
I think it's reasonable to just add `physicalID` to `regionMeta` and the `getRegionMeta` should fill the right value for it.  Good point

Reply:
Thanks, done!